rados/rados_getaddrs.go

```diff
@@ -0,0 +1,31 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package rados
+
+// #cgo LDFLAGS: -lrados
+// #include <stdlib.h>
+// #include <rados/librados.h>
+import "C"
+
+import (
+	"unsafe"
+)
+
+// GetAddrs wraps the function to get the addresses of
+// the RADOS session, suitable for blocklisting.
```

Comment:
Let's tell users what the function does not what it wraps. The Implements section handles telling them what it wraps, so:
```
GetAddrs returns a string containing the addresses of the RADOS session, suitable for blocklisting.
```
or similar.